docs/pages/management/operations/tls-routing.mdx

```diff
@@ -8,13 +8,15 @@ description: How to upgrade an existing Teleport cluster to single-port TLS rout
   opened={true}
   scope={["oss", "enterprise"]}
   scopeOnly={true}
-  min="8.0"
+  min="13.0"
 >
-  TLS routing is available starting from Teleport `8.0`.
+  TLS routing behind layer 7 (HTTP/HTTPS) load balancers and reverse proxies
+  support is available in Preview starting from Teleport `13.0`.
 </Details>
 
-Teleport supports TLS routing. In this mode, all client connections are wrapped
-in TLS and multiplexed on one Teleport proxy port.
+Starting from version `8.0`, Teleport supports TLS routing. In this mode, all
+client connections are wrapped in TLS and multiplexed on one Teleport proxy
+port.
```

Comment:
```suggestion
In TLS routing mode, all Teleport client connections are wrapped in TLS and multiplexed on one Teleport Proxy Service port.
```
- v8 is long out of support, so I don't think we need to mention it
- Suggested to make this more concise and capitalize "Proxy Service"